Servers/SystemServer/main.cpp

```diff
@@ -72,12 +81,26 @@ int main(int, char**)
 {
     int lowest_prio = sched_get_priority_min(SCHED_OTHER);
     int highest_prio = sched_get_priority_max(SCHED_OTHER);
-    start_process("/bin/LookupServer", lowest_prio);
-    start_process("/bin/WindowServer", highest_prio);
-    start_process("/bin/AudioServer", highest_prio);
-    start_process("/bin/Taskbar", highest_prio);
-    start_process("/bin/Terminal", highest_prio - 1);
-    start_process("/bin/Launcher", highest_prio);
+
+    // Mount the filesystems.
+    start_process("/bin/mount", "-a", highest_prio);
+    wait(nullptr);
+
+    start_process("/bin/TTYServer", "tty0", highest_prio, "/dev/tty0");
+    start_process("/bin/TTYServer", "tty1", highest_prio, "/dev/tty1");
+    start_process("/bin/TTYServer", "tty2", highest_prio, "/dev/tty2");
+    start_process("/bin/TTYServer", "tty3", highest_prio, "/dev/tty3");
+
+    // Drop privileges.
```

Comment:
This is okay for now, but in the future I think we might want to keep our privileges so we can do stuff like restarting a crashed `TTYServer`.
I guess an easy way to do this would be to drop privileges after forking.